.github/workflows/pr-realm-react.yml

```
@@ -0,0 +1,61 @@
+name: Testing Realm React (Pull Request)
+
+env:
+  REALM_DISABLE_ANALYTICS: 1
+
+on:
+  pull_request:
+    paths:
+      # Only run when the PR changes realm-react
+      - ".github/workflows/pr-realm-react.yml"
+      - "packages/realm-react/**"
+jobs:
+  job:
+    name: Build & test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: "recursive"
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 16
+          registry-url: https://registry.npmjs.org/
+
+      - name: Install npm v7
+        run: npm install -g npm@7
+
+      # Caches
+      - uses: actions/cache@v2
+        name: cached node_modules
+        id: cache-node-modules
+        with:
+          path: node_modules
+          key: node-modules-${{ github.job }}-${{ hashFiles('package-lock.json') }}
+
+      - name: ccache
+        uses: hendrikmuhs/ccache-action@v1
+        with:
+          key: realm-react
+
+      - name: Prepend ccache executables to the PATH
+        run: echo PATH="/usr/lib/ccache:/usr/local/opt/ccache/libexec:$PATH" >> $GITHUB_ENV
+
+      # Install the root package (--ignore-scripts to avoid downloading or building the native module)
+      - name: Install realm node dependencies
+        run: npm ci --ignore-scripts
```

```suggestion
        run: npm ci
```

I believe you could leave this out and remove the `npx prebuild` step as a result.